vumi/transports/vumi_bridge/vumi_bridge.py

```diff
@@ -134,6 +122,9 @@ def handle_outbound_message(self, message):
         if resp.code != http.OK:
             log.warning('Unexpected status code: %s, body: %s' % (
                 resp.code, resp.delivered_body))
+            self.update_status(
+                status='down', component='submitted-to-vumi-go',
+                type='bad_request', message='Bad request sent')
```

Comment:
Maybe the message here should be `Request rejected` (since we don't know if the request was bad -- only that Vumi Go didn't accept it).